AmplifyPlugins/Auth/AWSCognitoAuthPluginTests/AuthenticationProviderTests/AuthenticationProviderSigninTests.swift

```diff
@@ -12,8 +12,19 @@ import XCTest
 @testable import AWSCognitoAuthPlugin
 @testable import AWSMobileClient
 
+// swiftlint:disable file_length
+// swiftlint:disable type_body_length
 class AuthenticationProviderSigninTests: BaseAuthenticationProviderTest {
 
+    /// Test a signIn with valid inputs
+    ///
+    /// - Given: Given an auth plugin with mocked service.
+    ///
+    /// - When:
+    ///    - I invoke signIn with valid values
+    /// - Then:
+    ///    - I should get a .valid response
```

Comment:
Maybe something like: "I should get a valid response with .done"?

Reply:
Updated